cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -236,6 +236,13 @@ private static String findNetwork(String project, Compute compute) throws IOExce
       throw new IllegalArgumentException(String.format("Unable to find any networks in project '%s'. "
                                                          + "Please create a network in the project.", project));
     }
+
+    for (Network network: networks) {
+      if (network.getName().equals("default")) {
```

Comment:
don't think it's possible here, but it's generally safer to do:

`"default".equals(network.getName())`

as that protects against errors when the network name is null.

Reply:
Noted! Implemented the change.